### PR TITLE
Fix regressions caused by PR #1018

### PR DESCRIPTION
**Pull request by j-lum**

### CODE REVIEW

File: docs/SettingUp.adoc [hunk 1]
```diff
@@ -40,7 +40,7 @@ This will generate all resources required by the application and tests.
 == Verifying the setup
 
 . Run the `seedu.address.Main` and try a few commands
-. <<DeveloperGuide#Testing,Run the tests>> to ensure they all pass.
+. <<Testing#,Run the tests>> to ensure they all pass.
```
Comment (pyokagan) [inline docs/SettingUp.adoc]: Can remove the anchor in `DeveloperGuide.adoc` as well.

File: docs/SettingUp.adoc [hunk 2]
```diff
@@ -86,5 +85,6 @@ Having both Travis and AppVeyor ensures your App works on both Unix-based platfo
 
 When you are ready to start coding,
 
-1. Get some sense of the overall design by reading <<DeveloperGuide#Design-Architecture>>.
-2. Take a look at <<DeveloperGuide#GetStartedProgramming>>.
+1. Get some sense of the overall design by reading about
+<<DeveloperGuide#Design-Architecture, AddressBook's architecture>>.
+2. Take a look at <<DeveloperGuide#GetStartedProgramming, Appendix A, Suggested Programming Tasks to Get Started>>.
```
Comment (pyokagan) [inline docs/SettingUp.adoc]: Maybe remove the "Appendix A", otherwise it is an extra thing we need to remember to update if the appendix numbers change for some reason.

File: docs/SettingUp.adoc [hunk 3]
```diff
@@ -63,8 +63,7 @@ After forking the repo, the documentation will still have the SE-EDU branding an
 
 If you plan to develop this fork as a separate product (i.e. instead of contributing to `se-edu/addressbook-level4`), you should do the following:
 
-. Configure the <<Documentation#Docs-SiteWideDocSettings, site-wide documentation settings>> in link:{repoURL}/build
-.gradle[`build.gradle`], such as the `site-name`, to suit your own project.
+. Configure the <<Documentation#Docs-SiteWideDocSettings, site-wide documentation settings>> in link:{repoURL}/build.gradle[`build.gradle`], such as the `site-name`, to suit your own project.
```
Comment (pyokagan) [inline docs/SettingUp.adoc]: This should be in a different commit.

**Comment by j-lum** [comment]
Made the requested changes but the bot seems to be a bit busy.

**Comment by pyokagan** [comment]
> Made the requested changes but the bot seems to be a bit busy.

pls try again, i need to take a look at the Interdiff

File: docs/SettingUp.adoc [hunk 4]
```diff
@@ -85,5 +85,6 @@ Having both Travis and AppVeyor ensures your App works on both Unix-based platfo
 
 When you are ready to start coding,
 
-1. Get some sense of the overall design by reading <<DeveloperGuide#Design-Architecture>>.
-2. Take a look at <<DeveloperGuide#GetStartedProgramming>>.
+1. Get some sense of the overall design by reading about <<DeveloperGuide#Design-Architecture, AddressBook's
+architecture>>.
```
Comment (pyokagan) [inline docs/SettingUp.adoc]: Another extra newline added here. Asciidoc happens to render correctly (phew), but is it intended? I believe we don't usually add spurious newlines within a sentence.

File: docs/SettingUp.adoc (hunk 1 above)
Comment (pyokagan) [reply docs/SettingUp.adoc]: Did you miss this? https://github.com/se-edu/addressbook-level4/pull/1019#discussion_r301891992

Comment (j-lum) [reply docs/SettingUp.adoc]: I was under the impression that the anchor was required:
https://asciidoctor.org/docs/user-manual/#navigating-between-source-files

File: docs/SettingUp.adoc (hunk 4 above)
Comment (j-lum) [reply docs/SettingUp.adoc]: My IntelliJ seems to be wrapping lines occasionally (and erratically) right before committing.

File: docs/SettingUp.adoc (hunk 1 above)
Comment (pyokagan) [reply docs/SettingUp.adoc]: I meant this line: https://github.com/se-edu/addressbook-level4/blame/master/docs/DeveloperGuide.adoc#L264

Comment (j-lum) [reply docs/SettingUp.adoc]: Oh I misunderstood and thought you were referring to the exact #. Pushed a commit to address the block ID.
